app/src/marketplace/containers/Products/index.jsx

```diff
@@ -47,7 +47,7 @@ const Products = () => {
     const { api: createProductModal } = useModal('marketplace.createProduct')
 
     const loadCategories = useCallback(() => dispatch(getCategories(false)), [dispatch])
-    const { load: loadDataUnions, members } = useMemberStats()
+    const { load: loadDataUnions, members } = useAllDataUnionStats()
```

Comment:
The `useAllDataUnionStats` hook can be remove after DU 2.0 functionality is merged